cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2645,6 +2645,29 @@
     </description>
   </property>
 
+  <property>
+    <name>metrics.data.coarse.lag.factor</name>
+    <value>12</value>
+    <description>
+      Resolution-based factor that defines wall clock lag. After incoming metrics processing delay reaches this lag,
+      metrics will be aggregated to bigger inteval based on round factor. E.g. for resolution of 5 seconds and
```

Comment:
Are the second configured by the configuration constants? If so should we list them here?

Reply:
It should be in general metrics configuration documentation. It's impossible to document everything in each metrics configuration parameter. Here the value is used just to illustrate how factors work